packages/core/content-releases/package.json

```diff
@@ -85,6 +85,7 @@
     "typescript": "5.2.2"
   },
   "peerDependencies": {
+    "@strapi/admin": "^4.15.1",
```

Comment:
Is it enough to just list it as a peerDep? I still have an error in the ReleaseDetailsPage: '@strapi/admin' should be listed in the project's dependencies.